src/OoasUtil/ComLineProcesser.cs

```diff
@@ -132,6 +170,46 @@ public bool Process()
                             i++;
                             break;
 
+                        case "--derivedtypesreferencesforresponses":
```

Comment:
>derivedtypesreferencesforresponses [](start = 32, length = 34)

Shall we use Pascal format for mulitiple words?

Reply:
I was following the convention of the existing 'specversion' which does not use PascalCase.